src/Authentication/Controllers/RequestSystemUserController.cs

```diff
@@ -456,6 +456,34 @@ public async Task<ActionResult<RequestSystemResponse>> ApproveSystemUserRequest(
         return NotFound();
     }
 
+    /// <summary>
+    /// Escalates the Approval of the systemuser request, since the logged in user lack the AccessManager Role
+    /// The request is forwarded to the Portal where it will be visible for users with the AccessManager Role
+    /// </summary>
+    /// <param name="party">the partyId</param>
+    /// <param name="requestId">The UUID of the request to be approved</param>
+    /// <param name="cancellationToken">The cancellation token</param>
+    /// <returns>Status response model CreateRequestSystemUserResponse</returns>
+    [Authorize(Policy = AuthzConstants.POLICY_SCOPE_PORTAL)]
+    [Authorize(Policy = AuthzConstants.POLICY_ACCESS_MANAGEMENT_WRITE)]
+    [HttpPost("{party}/{requestId}/escalate")]
+    public async Task<ActionResult<bool>> EscalateApprovalSystemUserRequest(int party, Guid requestId, CancellationToken cancellationToken = default)
+    {
+        int userId = AuthenticationHelper.GetUserId(HttpContext);
+        Result<bool> response = await _requestSystemUser.EscalateApprovalSystemUser(requestId, party, userId, cancellationToken);
```

Comment:
What happens here when a DAGL or anyone with access to approve calls this endpoint? Shouldn't we validate?